packages/mars-theme/src/components/link.js

```diff
@@ -1,28 +1,19 @@
 import React from "react";
 import { connect } from "frontity";
 
-const Link = ({
-  actions,
-  libraries,
-  path,
-  page,
-  query,
-  className,
-  children
-}) => {
-  // If page is passed, build final href.
-  // const href = page ? `${path}page/${page}` : path;
-  const href = libraries.source.getRoute({ path, page, query });
+const Link = ({ actions, libraries, link, className, children }) => {
+  // normalizes link
+  link = libraries.source.stringify(link);
```

Comment:
Is this really needed?

Because it is used in `actions.router.set(link)` where `link` is normalized internally and in `href` where normalization is not important, right?

Reply:
You are right, it's not needed!